shared/utils.py

```
@@ -202,6 +202,19 @@ def open_folder(path: str) -> Tuple[bool, Optional[str]]:
         return False, f"Failed to open folder: {e}"
 
 
+def print_files(paths: List[str]) -> None:
+    """Send each file to the OS print handler (opens the system print dialog)."""
+    for path in paths:
+        if not os.path.isfile(path):
+            continue
+        if platform.system() == 'Windows':
+            os.startfile(path, 'print')  # type: ignore[attr-defined]
```

Ruff (0.15.10)

[error] Starting a process without a shell

(S606)